Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use the official Node.js:lst runtime as a base image
+FROM node:lts
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the build over
+COPY ./build/ ./app
+
+# Copy our dependencies
+COPY ./node_modules ./node_modules
```

Comment:
Do we need to worry about standardizing the way that node_modules is generated in case there are differences depending on the host environment? I was assuming we'd have a build image or something like that, but maybe for node it's not really needed